services/TikTok.json

```diff
@@ -14,6 +14,33 @@
       "filters": [
         "selectOnlyEuropeanPrivacyPolicy"
       ]
+    },
+    "communityGuidelines": {
+      "location": "https://www.tiktok.com/community-guidelines?lang=en",
+      "contentSelector": "article"
+    },
+    "lawEnforcementGuidelines": {
+      "location": "https://www.tiktok.com/legal/law-enforcement?lang=en",
+      "contentSelector": "article"
+    },
+    "copyrightPolicy": {
+      "location": "https://www.tiktok.com/legal/copyright-policy?lang=en",
+      "contentSelector": "article"
+    },
+    "virtualItemsPolicy": {
+      "location": "https://www.tiktok.com/legal/virtual-items?lang=en",
+      "contentSelector": "article",
+      "filters": [
+        "selectOnlyEuropeanVirtualItemsPolicy"
+      ]
+    },
+    "openSourceUse": {
```

Comment:
This is just a list of licenses for open-source software used in TikTok. I don't see the point in tracking those in CGUs 🙂

Reply:
Removed !